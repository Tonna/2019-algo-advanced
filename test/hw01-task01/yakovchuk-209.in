30
1 1 2 2 1 5 5 2 1 5 2 11 3 6 5 5 6 9 2 13 15 10 2 1 4 7 18 14 20
6
21 2
24 5
30 3
20 10
17 10
18 1
