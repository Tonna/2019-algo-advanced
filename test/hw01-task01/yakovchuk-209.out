16
9
42
20
27
6
